genesyscloud/outbound_campaign/resource_genesyscloud_outbound_campaign_schema.go

```diff
@@ -248,6 +248,12 @@ func ResourceOutboundCampaign() *schema.Resource {
 							Required:    true,
 							ForceNew:    true,
 						},
+						"filter": {
+							Description: "Whether to filter contacts dynamically.",
+							Type:        schema.TypeBool,
+							Required:    true,
```

Comment:
If this is now a required field, we need to add a schema migration that automatically updates existing orgs with the proper value in the state, along with instructions on how to update the config.

Reply:
I don't think it should be required if it didn't exist before